ui/app/controllers/feedback.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2021 Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import type Feedback from 'yavin-ui/models/feedback';
+import type NaviNotifications from 'navi-notifications/services/navi-notifications';
+
+export default class FeedbackController extends Controller {
+  declare model: Feedback;
+
+  @service
+  declare naviNotifications: NaviNotifications;
+
+  @action
+  setTitle({ target }: { target: HTMLInputElement }): void {
+    this.model.set('title', target?.value);
```

Comment:
I think this should work
```suggestion
    this.model.title = target?.value;
```